remote/rpc/doc.go

```diff
@@ -0,0 +1,28 @@
+package rpc
+
+/*
+
+This is a `net/rpc`-compatible implementation of a client and server
+for `flux/remote.Platform`.
+
+The purpose is to be able to access a daemon from an upstream
+service. The daemon makes an outbound connection (over, say,
+websockets), then the service can make RPC calls over that connection.
+
+On errors:
+
+Errors from the daemon can come in two varieties: application errors
+(i.e., a `*flux/errors.Error`), and internal errors (any other
```

Comment:
Nitpick: should probably be `*(flux/errors).Error` like how Go printifies em.

Reply:
Makes sense. I wonder if it is ambiguous otherwise, or just considered more clear.